website/package.json

```diff
@@ -15,7 +15,7 @@
     "crowdin-download": "crowdin-cli --config ../crowdin.yaml download -b master"
   },
   "dependencies": {
-    "docusaurus": "1.6.2",
+    "docusaurus": "1.7.2",
```

Comment:
let's bump the docusaurus version on a separate commit

Reply:
Sounds good. I tend to forget that these commits get squashed.